extension/src/popup/views/ManageAssets/index.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from "react";
+import { useSelector } from "react-redux";
+import { Redirect, Route, Switch } from "react-router-dom";
+
+import { transactionSubmissionSelector } from "popup/ducks/transactionSubmission";
+
+import { AddAsset } from "popup/components/manageAssets/AddAsset";
+import { ChooseAsset } from "popup/components/manageAssets/ChooseAsset";
+import { TrustlineError } from "popup/components/manageAssets/TrustlineError";
+
+import { PrivateKeyRoute } from "popup/Router";
+import { ROUTES } from "popup/constants/routes";
+
+export const ManageAssets = () => {
+  const { accountBalances } = useSelector(transactionSubmissionSelector);
+  const [errorAsset, setErrorAsset] = useState("");
+
+  const { balances } = accountBalances;
+
+  if (!balances) {
```

Comment:
nit: should we also disable the ManageAssets button if no assets? otherwise it will just load this view and then redirect back (or not sure if you were thinking of doing something different for next PR)

Reply:
We do actually hide the button based on the `isFunded` which, now that I'm thinking about it, means the same thing as `!balances`. I'll create a ticket to address that as `isFunded` shouldn't be needed